Dockerfile

```diff
@@ -1,10 +1,5 @@
-FROM node:14.18-alpine
-ENV APP_DIR=/usr/local/src/app
+FROM node:16-alpine
```

Comment:
Node 16 has already reached EOL. Is anything blocking us from going to the latest version of node?

Reply:
@sunu thanks for noticing! I've updated it to v18.